lib/cli/src/bin/wasmer.rs

```diff
@@ -1,5 +1,10 @@
 use wasmer_cli::cli::wasmer_main;
 
+#[cfg(not(any(feature = "cranelift", feature = "singlepass", feature = "llvm")))]
+compile_error!(
+    "Either enable at least one compiler, or compile the wasmer-headless binary instead"
+);
```

Comment:
This will make headless compilation fail.

We need to do:
```suggestion
#[cfg(not(any(feature = "cranelift", feature = "singlepass", feature = "llvm", feature = "headless-minimal")))]
compile_error!(
    "Either enable at least one compiler or use the headless-minimal feature"
);
```

Reply:
I think this file won't be compiled when compiling wasmer-headless. The CI failure came from the tests being executed without any flags on, I hope I fixed that.